qodana.yaml

```diff
@@ -8,6 +8,7 @@ profile:
 
 dotnet:
   solution: FluentAssertions.sln
+  frameworks: net47;net6.0;netstandard2.0;netstandard2.1
```

Comment:
I think we don't want .netstandard2.x to be reported

Reply:
Correct

Reply:
I guess we are close to a 0 error baseline report then :D

Reply:
Folks, need a little bit more time. We've implemented some kind of validation after the limitation feature was implemented. Gonna find the way how to supply the same config in a valid way.

Reply:
<img width="646" alt="image" src="https://github.com/fluentassertions/fluentassertions/assets/48032702/aeb22d41-d313-4cfa-bbe7-83c9e22d1544">
<img width="655" alt="image" src="https://github.com/fluentassertions/fluentassertions/assets/48032702/a864b7b8-7fd9-4721-b601-cb6c36aef461">

Reply:
<img width="653" alt="image" src="https://github.com/fluentassertions/fluentassertions/assets/48032702/7f9bc841-14c7-4bc9-a5ed-f63bcdea6f13">

Reply:
@dennisdoomen the last configuration works (with plus list). We will think how we could supply the minus list then for 241.